substrate/frame/session/benchmarking/src/mock.rs

```diff
@@ -42,6 +42,7 @@ frame_support::construct_runtime!(
 		Balances: pallet_balances,
 		Staking: pallet_staking,
 		Session: pallet_session,
+		Historical: pallet_session::historical
 	}
```

Comment:
It seems that benchmark v1 did not notice that the code doesn't compile for the extra.